components/quick_switch_modal/quick_switch_modal.tsx

```diff
@@ -317,6 +317,7 @@ export default class QuickSwitchModal extends React.PureComponent<Props, State>
                         <i className='icon icon-magnify icon-16'/>
                         <SuggestionBox
                             id='quickSwitchInput'
+                            aria-label='quick switch input'
```

Comment:
Hello @devinbinnie, I added you as a reviewer especially for this added `aria-label`.

Reply:
Might want this to be a translatable string.

Reply:
Done, please take a look again. Thanks!